docs/reference/api.rst

```diff
@@ -158,6 +163,13 @@ Modules
 
 .. autodata:: inmanta.module.Path
 
+.. autoclass:: inmanta.loader.PluginModuleFinder
+    :show-inheritance:
+    :members: configure_module_finder, reset
```

Comment:
Why add `configure_module_finder` to stable API?

Reply:
This seems to be a mistake. I was under the impression it was already used by pytest-inmanta but it turns out to not be the case. Thanks.